src/sentry/api/serializers/models/projectcodeowners.py

```diff
@@ -33,4 +43,11 @@ def serialize(self, obj, attrs, user):
 
         data["provider"] = attrs.get("provider", "unknown")
 
+        if "codeMapping" in self.expand:
+            config = attrs.get("codeMapping", {})
+            data["codeMapping"] = serialize(
+                config, user=user, serializer=RepositoryProjectPathConfigSerializer()
+            )
+            data.pop("codeMappingId", None)
```

Comment:
I actually think this might be good to keep, I know it's it's kind of redundant but I think it's better to let people always rely on the `codeMappingId` be there whether or not we add the full `codeMapping`